### DISCUSSION TITLE
Question about translation

### DISCUSSION THREAD

**Discussion by azzozhsn** [statamic/cms#4825] [Q&A]
what is the correct path for translation for the following key:

statamic::messages.getting_started_widget_intro

the part 'statamic::' confuses me

**Comment by jasonvarga** [comment]
The `statamic::` prefix means it's in the `statamic` "namespace".
This means they'd load our translations, which live in `vendor/statamic/cms/resources/lang/en/messages.php`.
If you want to override them, you should place your own files in `resources/lang/vendor/statamic/en/messages.php`

**Comment by azzozhsn** [comment]
So, I can not add language without forking the repo, right?
I mean I can not add language to my project directly, right?

**Reply by jasonvarga** [reply]
You can put it in `resources/lang/vendor/statamic/the_new_language`

But if you are translating the control panel into a language we don't already support, perhaps you'd consider contributing it so it could be available for others.
